config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-gcr-to-ar-sync.yaml

```diff
@@ -1,31 +1,33 @@
 # If you are adding new AR regions, you need to run this job for a while and then add the manifests in k/k8s.io
-# periodics:
-#   - name: gcr-to-ar-sync-europe
-#     interval: 2h
-#     cluster: k8s-infra-prow-build-trusted
-#     decorate: true
-#     max_concurrency: 1
-#     annotations:
-#       testgrid-dashboards: sig-k8s-infra-k8sio
-#       testgrid-alert-email: k8s-infra-alerts@kubernetes.io
-#       testgrid-num-failures-to-alert: '1'
-#     rerun_auth_config:
-#       github_team_slugs:
-#         - org: kubernetes
-#           slug: sig-k8s-infra-leads
-#         - org: kubernetes
-#           slug: release-managers
-#     spec:
-#       serviceAccountName: k8s-infra-gcr-promoter
-#       containers:
-#         - image: golang:1.18
-#           imagePullPolicy: Always
-#           command:
-#             - /bin/bash
-#           args:
-#             - -c
-#             - |
-#               export PATH=$PATH:$GOPATH/bin
-#               apt update && apt install parallel -qqy
-#               go install github.com/google/go-containerregistry/cmd/gcrane@latest
-#               parallel "gcrane cp --recursive --allow-nondistributable-artifacts eu.gcr.io/k8s-artifacts-prod {}-docker.pkg.dev/k8s-artifacts-prod/images" ::: europe-north1 europe-southwest1 europe-west1 europe-west2 europe-west4 europe-west8 europe-west9
+periodics:
+  - name: gcr-to-ar-sync-us
+    cron: "0 */2 * * *" # 2h
+    # cron: "0 0 31 4 *" # 31st of April
+    cluster: k8s-infra-prow-build-trusted
+    decorate: true
+    max_concurrency: 1
+    annotations:
+      testgrid-dashboards: sig-k8s-infra-k8sio
+      testgrid-alert-email: k8s-infra-alerts@kubernetes.io
+      testgrid-num-failures-to-alert: '1'
+    rerun_auth_config:
+      github_team_slugs:
+        - org: kubernetes
+          slug: sig-k8s-infra-leads
+        - org: kubernetes
+          slug: release-managers
+    spec:
+      serviceAccountName: k8s-infra-gcr-promoter
+      containers:
+        - image: golang:1.19
+          imagePullPolicy: Always
+          command:
+            - /bin/bash
+          args:
+            - -c
+            - |
+              export PATH=$PATH:$GOPATH/bin
+              apt update && apt install parallel -qqy
+              go install github.com/google/go-containerregistry/cmd/gcrane@latest
+              parallel "gcrane cp --recursive --allow-nondistributable-artifacts us.gcr.io/k8s-artifacts-prod {}-docker.pkg.dev/k8s-artifacts-prod/images" ::: us-west3 us-west4
```

Comment:
Maybe it's too late but I feel like k8s-artifacts-prod is too welcoming of end-user usage.

I guess we can restrict read access to this registry? Or we could still rename it to something something internal-do-not-use ?

Reply:
We don't need to rename and we can skip adding IAM bindings to the bucket. The principals that need to read/write should be granted permissions on the project.

Reply:
If `allUsers` doesn't have `storage.objectViewer` role on the bucket, it will be return 403 when someone on the internet tries to pull an image from `gcr.io/k8s-artifacts-prod`

Reply:
ah right, I missed that we're not opening this to allUsers like the others, SGTM